docs/sources/flow: various documentation tweaks (#2220)
grafana/agent: docs/sources/flow/reference/cli/run.md

* `--disable-reporting`: Disable [usage reporting][] of enabled [components][] to Grafana (default `false`).

[usage reporting]: {{< relref "../../../configuration/flags.md/#report-information-usage" >}}
[components]: {{< relref "../../concepts/components.md" >}}

## Updating the config file

The config file can be reloaded from disk by either:

* Sending an HTTP POST request to the `/-/reload` endpoint.
* Sending a `SIGHUP` signal to the Grafana Agent process.

> When this happens, the [component controller][] will synchronize the set of

Reviewer: ```suggestion
When this happens, the [component controller][] synchronizes the set of
```

running components with the latest set of components specified in the config

> file. Components which are no longer defined in the config file after reloading

Reviewer: ```suggestion
file. Components that are no longer defined in the config file after reloading
```

> will be shut down, and components which have been added to the config file

Reviewer: ```suggestion
are shut down, and components that have been added to the config file
```

> since the previous reload will be created.

Reviewer: ```suggestion
since the previous reload are created.
```
